src/main/java/com/kubetetris/interaction/KubernetesAccessorImplV2.java

```diff
@@ -0,0 +1,204 @@
+package com.kubetetris.interaction;
+
+import com.kubetetris.Node;
+import com.kubetetris.Pod;
+import com.kubetetris.Resource;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.*;
+import io.kubernetes.client.proto.V1;
+import io.kubernetes.client.util.Config;
+import lombok.extern.slf4j.Slf4j;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.ParseException;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class KubernetesAccessorImplV2 implements KubernetesAccessor{
+
+    private static long DEFAULT_CPU_HEADROOM = 5000;
+    private static long DEFAULT_MEMORY_HEADROOM = 5000;
+
+    CoreV1Api api;
+
+    {
+        try {
+            ApiClient client = Config.fromConfig("/Users/yashbhatnagar/Downloads/kubeconfig");
```

Comment:
Let's not hardcode this and instead have this as a configurable property. It will fail if anyone tries to use this.